src/main/resources/db/migration/changes/UCSBOrganization.json

```diff
@@ -25,21 +25,21 @@
                     "column": {
                       "constraints": {
                         "primaryKey": true,
-                        "primaryKeyName": "CONSTRAINT_27"
+                        "primaryKeyName": "CONSTRAINT_69"
```

Comment:
im not sure if this can be named "CONSTRAINT_69" because i named it 😭 but im actually not too sure, ill just leave this here as a note